cloudigrade/api/clouds/aws/tasks/inspection.py

```diff
@@ -2,578 +2,111 @@
+        LaunchTemplate={
+            "LaunchTemplateName": f"cloudigrade-lt-{settings.CLOUDIGRADE_ENVIRONMENT}"
         },
-    )
-    ecs.run_task(
-        cluster=settings.HOUNDIGRADE_ECS_CLUSTER_NAME,
-        taskDefinition=task_definition_arn,
+        MaxCount=1,
+        MinCount=1,
+        UserData=cloud_init_script,
     )
 
 
-def _build_container_definition(task_command):
+def _build_cloud_init_script(ami_id):
     """
-    Build a container definition to be used by an ecs task.
+    Build a cloud init script that'll run the inspection.
 
     Args:
-        task_command (list): Command to insert into the definition.
+        ami_id(str): ID of the AMI being inspected
 
-    Returns (dict): Complete container definition.
+    Returns (str): Complete cloud init script.
 
     """
-    container_definition = {
-        "name": "Houndigrade",
-        "image": f"{settings.HOUNDIGRADE_ECS_IMAGE_NAME}:"
-        f"{settings.HOUNDIGRADE_ECS_IMAGE_TAG}",
-        "cpu": 0,
-        "memoryReservation": 256,
-        "essential": True,
-        "command": task_command,
-        "environment": [
-            {"name": "AWS_DEFAULT_REGION", "value": settings.AWS_SQS_REGION},
-            {"name": "AWS_ACCESS_KEY_ID", "value": settings.AWS_SQS_ACCESS_KEY_ID},
-            {
-                "name": "AWS_SECRET_ACCESS_KEY",
-                "value": settings.AWS_SQS_SECRET_ACCESS_KEY,
-            },
-            {
-                "name": "RESULTS_QUEUE_NAME",
-                "value": settings.HOUNDIGRADE_RESULTS_QUEUE_NAME,
-            },
-            {"name": "EXCHANGE_NAME", "value": settings.HOUNDIGRADE_EXCHANGE_NAME},
-            {"name": "QUEUE_CONNECTION_URL", "value": settings.CELERY_BROKER_URL},
-        ],
-        "privileged": True,
-        "logConfiguration": {
-            "logDriver": "awslogs",
-            "options": {
-                "awslogs-create-group": "true",
-                "awslogs-group": f"{settings.AWS_NAME_PREFIX}cloudigrade-ecs",
-                "awslogs-region": settings.AWS_SQS_REGION,
-            },
-        },
-    }
-    if settings.HOUNDIGRADE_ENABLE_SENTRY:
-        container_definition["environment"].extend(
-            [
-                {
-                    "name": "HOUNDIGRADE_SENTRY_DSN",
-                    "value": settings.HOUNDIGRADE_SENTRY_DSN,
-                },
-                {
-                    "name": "HOUNDIGRADE_SENTRY_RELEASE",
-                    "value": settings.HOUNDIGRADE_SENTRY_RELEASE,
-                },
-                {
-                    "name": "HOUNDIGRADE_SENTRY_ENVIRONMENT",
-                    "value": settings.HOUNDIGRADE_SENTRY_ENVIRONMENT,
-                },
-            ]
-        )
+    houndigrade_image = (
+        f"{settings.HOUNDIGRADE_ECS_IMAGE_NAME}:{settings.HOUNDIGRADE_ECS_IMAGE_TAG}"
+    )
 
-    return container_definition
+    cloud_init_script = f"""Content-Type: multipart/mixed; boundary="//"
```

Comment:
Not something you need to change here now, but here's a tip that I've used elsewhere for heredocs or long strings. Since the string definition here removed the normal indentation to avoid unwanted spaces in the output, the tradeoff is a reduction in the readability and broken visual flow of the surrounding Python code. Fortunately, Python has a function to help with this kind of situation!

```python
>>> name = "potato"
>>> foo = f"""
...        hello {name}
...        this string has
...        lots of leading spaces!
...        """
>>> print(foo)

       hello potato
       this string has
       lots of leading spaces!

>>> from textwrap import dedent
>>> print(dedent(foo))

hello potato
this string has
lots of leading spaces!
```

Unfortunately, `dedent` requires _all_ not-empty lines to be indented to have the desired effect. So, in this specific code where `cloud_init_script` starts with `Content-Type` on the first line, it wouldn't work unless you moved that content down to the next line.